tools/boundary/glorys_obc_workflow/mom6_obc_workflow.sh

```diff
@@ -0,0 +1,178 @@
+#!/bin/bash
+set -eu
```

Comment:
This section keeps giving me the following error: 
```
/app/conda/miniconda/etc/conda/deactivate.d/udunits2-deactivate.sh: line 5: _CONDA_SET_UDUNITS2_XML_PATH: unbound variable
```
It looks like this is an [open issue](https://github.com/conda/conda/issues/9966) that occurs when you try to activate conda envs with `set -u`. The issue page suggests get around this by moving the set -eu line after the conda activate line, which worked for me.